src/Facility.CodeGen.JavaScript/JavaScriptGenerator.cs

```diff
@@ -42,7 +42,9 @@ protected override CodeGenOutput GenerateOutputCore(ServiceInfo service)
 			string serverFileName = Uncapitalize(moduleName) + "Server" + (TypeScript ? ".ts" : ".js");
 
 			var namedTexts = new List<CodeGenFile>();
-			var typeNames = new List<string>();
+			var requestTypeNames = new List<string>();
+			var responseTypeNames = new List<string>();
+			var dtoTypeNames = new List<string>();
```

Comment:
I broke up the types and made this before I realized this wasn't actually being used anywhere.